test/e2e/testing-manifests/kubectl/nginx-statefulset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
```

Comment:
@janetkuo the validation codes require `serviceName` to be specified in the spec. i think we should include the service in the YAML file, create it, and then delete the service after the statefulset is deleted.